app/graphql/types/activity_type.rb

```diff
@@ -3,6 +3,7 @@ class ActivityType < Types::BaseObject
     field :id, ID, null: false
     field :title, String, null: false
     field :body, String, null: false
+    field :event, String, null: false
```

Comment:
GraphQL has enum type. Would it be reasonable to use it here?
```suggestion
    field :event, ActivityEventStatusType, null: false
```

```
module Types
  class ActivityEventStatusType < Types::BaseEnum
    value "USER_REGISTRED", value: "user_registered"
  end
end

```